docs/administration-guide/databases/mysql.md

```diff
@@ -71,3 +71,9 @@ mysql:
 The problem is that if the keys in the JSON vary record to record, the first five hundred rows may not capture all the keys used by JSON objects in that JSON field. To get Metabase to infer all the JSON keys, you'll need to add the additional keys to the JSON objects in the first five hundred rows.
 
 2. **This JSON support doesn't work with MariaDB**, due to implementation differences between MySQL and MariaDB.
+
+## Model caching
+
+We'll create tables with model data and refresh them on a schedule you define. To enable it, you need to grant this connection's credentials read and write permissions for the schema displayed in the info tooltip, or grant create schema permissions.
+
```

Comment:
"you need to grant this connection's credentials read and write permissions" . The way this is phrased makes me think its action in Metabase. 
> The connection's credentials must be able to read and write to the schema displayed in the info tooltip

Maybe not that exact phrasing but inverting the action maybe makes it clearer that its a condition necessary for use, and not an action to be taken in Metabase?

Nitpick and if you don't agree feel free to ignore. Just something that caught my eye on first read.

Reply:
I took that phrase from the in-product copy. I agree it's not clear. I can fix up the copy in the docs, and maybe the tooltip as well.